packages/azure-kusto-data/source/errors.ts

```diff
@@ -1,9 +1,18 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+/* eslint max-classes-per-file: ["error", 2] */
```

Comment:
We usually disable it with an explanation like so:
/* eslint-disable max-classes-per-file -- We want all the Token Providers in this file */

I am considering on removing this lint entirely later